src/briefcase/console.py

```diff
@@ -399,7 +412,8 @@ def _build_log(self, command):
         # build log header and export buffered log from Rich
         uname = platform.uname()
         sanitized_env_vars = "\n".join(
-            f"    {env_var}={value if not SENSITIVE_SETTING_RE.search(env_var) else '********************'}"
+            f"\t{env_var}="
+            f"{sanitize_text(value) if not SENSITIVE_SETTING_RE.search(env_var) else '********************'}"
```

Comment:
Is this advisable? Environment variables are something that we really need to know verbatim - any manipulation could lead us to believe that one value has been set, when it's actually a different value. 

To be clear - I can't think of a reason why the user's PS1 setting could cause a problem, but then, that's likely a lack of imagination on my part.

Reply:
Yeah; that's fair....and it's really of marginal value to remove these anyway.